src/lhttpc.app.src

```diff
@@ -30,11 +30,13 @@
 %%% @end
 {application, lhttpc,
     [{description, "Lightweight HTTP Client"},
-        {vsn, "1.3.0"},
+        {vsn, git},
         {modules, []},
         {registered, [lhttpc_manager]},
         {applications, [kernel, stdlib, ssl, crypto]},
   {mod, {lhttpc, nil}},
-  {env, [{connection_timeout, 300000}, {pool_size, 50}]}
+  %% make default pool ~ all possible ports for a single IP interface. e.g. Usual 1025-65535
+  %% define your pool or other size if you want to limit it or have more IP interfaces
+  {env, [{connection_timeout, 300000}, {pool_size, 64000}]}
```

Comment:
motivation for the change is that I already lost count of how many times people faced an issue of unexpected client failures due to extremely small pool where it always came by surprise that there is any limit at all. 
One may say "read the source" and there will be no issues but i feel failing with error on tcp open if more friendly and clearly indicates the issue to the consumer.